force-app/main/default/classes/ADDR_Addresses_UTIL.cls

```diff
@@ -28,14 +28,13 @@
     POSSIBILITY OF SUCH DAMAGE.
 */
 /**
-* @author Salesforce.org 
-* @date 2016 
-* @group Addresses
-* @group-content ../../ApexDocContent/Addresses.htm
-* @description Addresses utility methods.
-*/
+ * @author Salesforce.org
+ * @date 2016
+ * @group Addresses
+ * @group-content ../../ApexDocContent/Addresses.htm
+ * @description Addresses utility methods.
+ */
 public with sharing class ADDR_Addresses_UTIL {
```

Comment:
Really happy to see this getting prettier applied to it so that meaningful changes are captured in other PRs and aren't lost in the shuffle on this one.